docker-compose.yml

```diff
@@ -1,16 +1,25 @@
-version: '3.1'
+version: "3.5"
```

Comment:
Apparently to use a named network you need to use version 3.5.  See https://stackoverflow.com/a/48024244